src/main/java/net/twasi/obsremotejava/message/request/inputs/SetInputVolumeRequest.java

```diff
@@ -0,0 +1,29 @@
+package net.twasi.obsremotejava.message.request.inputs;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+import net.twasi.obsremotejava.message.request.Request;
+
+@Getter
+@ToString(callSuper = true)
+public class SetInputVolumeRequest extends Request {
+    private final Data requestData;
+
+    @Builder
+    private SetInputVolumeRequest(float inputVolumeDb, float inputVolumeMul) {
```

Comment:
@ChristopheCVB Was thinking for shortened stuff like this that we could give it a longer name so devs can more-easily tell what it's for, but then add the serialization annotation so it is sent over the wire correctly. 

What do you think?

Reply:
I think that we should be as much transparent as possible and use the same names described in doc

Reply:
Agreed! We should favor transparency, I guess we'd only consider changing it if enough people ask for it, if that.